ChatApp/Controlers/Themeable.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ViewTheme.swift
+//  ChatApp
+//
+//  Created by bakhva  on 23.05.23.
+//
+
+import Foundation
+import UIKit
+
+protocol Themeable {
+    func isDarkModeOn(isTrue: Bool)
```

Comment:
isTrue is kind of extra when calling the function

This will be better:
```suggestion
    func isDarkModeOn(_ isTrue: Bool)
```